internal/db/audit.go

```diff
@@ -98,8 +98,12 @@ func (d *Database) ForEachAuditLogEntry(ctx context.Context, filter AuditLogFilt
 	if filter.SortTime {
 		db = db.Order("time DESC")
 	}
-	db = db.Limit(filter.Limit)
-	db = db.Offset(filter.Offset)
+	if filter.Limit > 0 {
```

Comment:
in this way if somebody set it to 0 (or default zero-valued) it gets all the records. 
Here, which is the rebac_admin entrypoint, we set guards for max number of elements and we use "-1" for the default page size.
https://github.com/canonical/jimm/blob/ee16e51a426683210a6aa5b25459e03114fffdab/internal/common/pagination/pagination.go#L22-L36

Reply:
i don't quite understand. If 0, all records should be returned. If somebody wants to limit the number of records returned they should set a limit that is higher than 0. Is that not correct?

Reply:
we have two entrypoints for groups:
- jimmctl
- rebac_admin

For the rebac_admin implementation there is no way to return all records, we have a default and max page size.

Here with the jimmctl we are not doing that, we are defaulting to returning all records.
Is that the correct approach?

Reply:
yes

Reply:
Shouldn't we guard for a max size? I genuinely don't know what's the best approach to display 1000 elements in a CLI tool.
Pagination doesn't make much sense, but also returning many many records doesn't seem very smart.

Reply:
i am not talking about groups, but more in general for all of our outputs

Reply:
but then we need to change jimmctl commands, as well.. followup as it exceeds the scope of this PR